subsys/dfu/Kconfig

```
@@ -33,6 +33,7 @@ config DFU_TARGET_MCUBOOT_SAVE_PROGRESS
 
 config DFU_TARGET_MODEM
 	bool "Modem update support"
+	imply DOWNLOAD_CLIENT_RANGE_REQUESTS
```

Not a blocker, but you should change it if you are already doing changes.

With `imply` it is still possible for the user to disable the `DOWNLOAD_CLIENT_RANGE_REQUESTS` while enabling `DFU_TARGET_MODEM` and you will not get an error from Kconfig.

```suggestion
	depends on DOWNLOAD_CLIENT_RANGE_REQUESTS
```